cvxpygen/cpg.py

```diff
@@ -256,8 +265,8 @@ def user_p_value(user_p_id):
         n_eq = data['n_eq']
         n_ineq = data['n_ineq']
 
-        indices_obj, indptr_obj, shape_obj = p_prob.problem_data_index_P
-        indices_constr, indptr_constr, shape_constr = p_prob.problem_data_index_A
+        indices_obj, indptr_obj, shape_obj = p_prob.reduced_P.problem_data_index
+        indices_constr, indptr_constr, shape_constr = p_prob.reduced_A.problem_data_index
```

Comment:
Out of curiousity is this part of the public API for CVXPY? Like should this work in 1.4

Reply:
No this is not public API. I don't mind if it breaks cvxpygen every minor release. It's easy enough to fix.